utilities/formateDate.js

```diff
@@ -0,0 +1,21 @@
+export function formatDate(x){
+    
+    const past = new Date(x);
+    const now = new Date();
+
+    const result = now - past;
+    const seconds = Math.floor(result / 1000);
```

Comment:
the date object has methods to get seconds, minutes, hours and days. use those instead

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDay